test/01_basic/cibuildwheel_test.py

```diff
@@ -19,4 +19,5 @@ def test_build_identifiers():
     # identifiers
     expected_wheels = utils.expected_wheels('spam', '0.1.0')
     build_identifiers = utils.cibuildwheel_get_build_identifiers(project_dir)
+    print("bb", build_identifiers)
```

Comment:
What's this, still?

Reply:
Sorry.

Reply:
No problem. Very easily overlooked. I was just lucky to notice it ;-)